test/parallel/test-repl-definecommand.js

```diff
@@ -23,22 +23,28 @@ r.defineCommand('say1', {
   help: 'help for say1',
   action: function(thing) {
     output = '';
-    this.write(`hello ${thing}`);
+    this.output.write(`hello ${thing}\n`);
     this.displayPrompt();
   }
 });
 
 r.defineCommand('say2', function() {
   output = '';
-  this.write('hello from say2');
+  this.output.write('hello from say2\n');
   this.displayPrompt();
 });
 
 inputStream.write('.help\n');
-assert(/\n\.say1     help for say1\n/.test(output),
-       'help for say1 not present');
+assert.ok(/\n\.say1     help for say1\n/.test(output),
+          'help for say1 not present');
```

Comment:
This seems to be a independent change. I do not really see why this should be changed?

Reply:
I was trying to standardize on `assert.ok()` for tests, but I see I also missed one on another line. Anyway, yes, it's unnecessary, so I'll undo it.